specs/glossary.md

```diff
@@ -367,6 +368,14 @@ block number or a given block hash.
 
 cf. [L1 Attributes Predeployed Contract Specification](deposits.md#l1-attributes-predeployed-contract)
 
+## L2 Output Root
+
+[l2-output]: glossary.md#l2-output
+
+A 32 byte value representing the current state of the L2 chain.
```

Comment:
I think that the phrase "commits to" is better than "representing", ie "A 32 byte value that commits to the current state of the L2 chain"

And I believe that we did not opt to have it commit to more values than just the state trie? We should be clear here with what exactly it commits to.

Also is there a versioning scheme? ie so that it can be updated to verkle trees easily or commit to a merkle tree that includes multiple interesting things like @protolambda has suggested

Reply:
It will initially commit to:
- the block
- the state trie
- the storage root of the withdrawals predeploy
- the version

There is a more complex output [currently specified](https://github.com/ethereum-optimism/optimistic-specs/blob/429079094899becdf4d37921938650d2f8d8b139/specs/proposals.md#l2-output-commitment-construction), but I plan to simplify it to the above in #253.